api/module/ModuleRouter.go

```diff
@@ -0,0 +1,22 @@
+package module
+
+import (
+	"github.com/gorilla/mux"
+)
+
+type ModuleRouter interface {
+	Init(configRouter *mux.Router)
+}
+
+type ModuleRouterImpl struct {
+	moduleRestHandler ModuleRestHandler
+}
+
+func NewModuleRouterImpl(moduleRestHandler ModuleRestHandler) *ModuleRouterImpl {
+	return &ModuleRouterImpl{moduleRestHandler: moduleRestHandler}
+}
+
+func (impl ModuleRouterImpl) Init(configRouter *mux.Router) {
+	configRouter.Path("").HandlerFunc(impl.moduleRestHandler.GetModuleInfo).Queries("name", "{name}").Methods("GET")
+	configRouter.Path("").HandlerFunc(impl.moduleRestHandler.HandleModuleAction).Queries("name", "{name}").Methods("POST")
```

Comment:
can we take "name" in request payload instead of query param in POST method.

Reply:
yes, we can do that, but that would require change on FE also. 
Should we still do that?